systemtest/rum_test.go

```diff
@@ -65,7 +66,23 @@ func TestRUMXForwardedFor(t *testing.T) {
 		"@timestamp", "timestamp.us",
 		// RUM events have the source port recorded, and in the tests it will be dynamic
 		"source.port",
+		// Do not assert the exact contents of the geo field since they can change slightly
+		// depending on the IP lookup.
+		"client.geo",
```

Comment:
```suggestion
		// Do not assert the exact contents of the geo field since they can change slightly
		// depending on the IP lookup.
		"client.geo.location",
```

IMO we should either only disregard `client.geo.location`, or not use approvals at all. The test is only about making sure X-Forwarded-For makes it through to geoIP enrichment.

Reply:
I'll amend that. I initially went that route but then thought we might not want to assert the contents at all, I'll revert to what I asserted initially.